### ISSUE TITLE
Develop apps into chrome as a chrome extension

### ISSUE THREAD

**Issue by clearloop** [vue-polkadot/apps#66]
## Spec

Using `vue-polkadot` just like `metamask`.

1. `vue-polkadot/apps`'s functions are duplicate to `polkadot/apps`, just different implementation in some way, and we can do it better.
2. `apps` has wallet functions, this makes `vue-polkadot` has the potential to be a `metamask` in substrate ecologies
3. Chrome extensions are better than web pages...(you know, I input the link to my browser every time I'm trying to open the `apps`...)
4. A nice wallet will reach every chain which is built by substrate, once we develop Kota into chrome, things will change, and substrate will be stronger because of Kota

**Comment by yangwao** [comment]
@clearloop I clearly see nice ideas here, I'm really glad for them! 🤗

> vue-polkadot/apps's functions are duplicate to polkadot/apps, just different implementation in some way, and we can do it better.

We want to be somehow performance, mobile-first and lightweight version, just focused version

> apps has wallet functions, this makes vue-polkadot has the potential to be a metamask 
in substrate ecologies

Yes, we are using keyring for accounts/addressbook and transfer, that could be potentially easy exportable part.

> Chrome extensions are better than web pages...(you know, I input the link to my browser every time I'm trying to open the apps...)

Yes, but it's somehow vendor-lockin right now. We played little bit with handler when you want to send from a particular address, so we are thinking to make URI for that.

> A nice wallet will reach every chain which is built by substrate, once we develop Kota into chrome, things will change, and substrate will be stronger because of Kota

Kota was also part of the naming part 😁. We definitely see this as a vector of opportunities, yet we are not only one, but only one who covers most functionalities so far.

We've recently [published article about kodadot development](https://medium.com/kodadot/kodadot-return-of-the-vue-js-b9ec01268ea2) and we continue to deliver other features and optimizations. UX is in first place where we want to focus if the working environment allows us.

We are definitely looking for cooperation, we recently extended to another explorer like Subscan.io for next release

**Comment by clearloop** [comment]
> We want to be somehow performance, mobile-first and lightweight version, just focused version

I love the mobile-first direction, btw, is there any telegram group or somewhere we can chat about Kota? I'd like to talk about it when I have some thoughts.

**Comment by yangwao** [comment]
> I love the mobile-first direction, btw, is there any telegram group or somewhere we can chat about Kota? I'd like to talk about it when I have some thoughts.

Yes, we have issues https://github.com/vue-polkadot/apps/issues/58 and https://github.com/vue-polkadot/apps/issues/46 regarding mobile app. Buefy recently released sidebar component, which probably should fix some issues, @vikiival will be working on that. We will see whenever it will be done.

Right now the best spot is to open issue. 
We don't have capacity to maintain telegram channel.

**Comment by vikiival** [comment]
Sidebar has currently [small bug](https://github.com/buefy/buefy/issues/2466)

**Comment by vikiival** [comment]
Sidebar fixed in #75
